### PR TITLE
Read Captcha Settings From ENV

### PR DESCRIPTION
**Pull request by FatihKoz**

### CODE REVIEW

File: config/captcha.php [hunk 1]
```diff
@@ -7,17 +7,17 @@
  * Don't edit this file directly, add the section to your config.php
  */
 return [
-    'enabled' => false,
-    'sitekey' => '',
-    'secret'  => '',
+    'enabled' => env('CAPTCHA_ENABLED', 'false'),
```
Comment (nabeelio) [inline config/captcha.php:10]: Crap, merged this too fast. The `'false'` shouldn't be in quotes.

Comment (FatihKoz) [reply config/captcha.php:10]: it will always be enabled then ?

Comment (nabeelio) [reply config/captcha.php:10]: No, it should still evaluate to false

Comment (FatihKoz) [reply config/captcha.php:10]: it was kind of a harmless mistake then... Fixed ;)

**Pull request merged**